toctocbin/AndroidArchitecture/app/src/main/java/com/test/androidarchitecture/ui/market/MarketActivity.kt

```diff
@@ -6,57 +6,36 @@ import android.widget.Toast
 import com.test.androidarchitecture.R
 import com.test.androidarchitecture.adpter.ViewPagerAdapter
 import com.test.androidarchitecture.data.MarketTitle
-import com.test.androidarchitecture.data.source.UpbitRepository
-import io.reactivex.android.schedulers.AndroidSchedulers
-import io.reactivex.disposables.CompositeDisposable
 import kotlinx.android.synthetic.main.activity_market.*
 
-class MarketActivity : AppCompatActivity() {
+class MarketActivity : AppCompatActivity(), MarketContract.View {
```

Comment:
슈퍼클래스와 인터페이스는 개행해서 작성해주는것에 좋겠네요

https://kotlinlang.org/docs/reference/coding-conventions.html#class-header-formatting

```suggestion
class MarketActivity : 
    AppCompatActivity(), 
    MarketContract.View {
```

Reply:
ca8ddcf

변경했습니다.